ibis/backends/tests/test_array.py

```diff
@@ -1478,3 +1471,104 @@ def test_table_unnest_array_of_struct_of_array(con):
     result = con.execute(expr)
     expected = pd.DataFrame({"x": [1, 1, 1, 2, 2, 2, 3, 3, 3]})
     tm.assert_frame_equal(result, expected)
+
+
+notimpl_aggs = pytest.mark.notimpl(
+    ["datafusion", "flink", "pandas", "dask"], raises=com.OperationNotDefinedError
+)
+
+
+def _agg_with_nulls(agg, x):
+    if x is None:
+        return None
+    x = [y for y in x if not pd.isna(y)]
+    if not x:
+        return None
+    return agg(x)
+
+
+@pytest.mark.parametrize(
+    ("agg", "baseline_func"),
+    [
+        (ir.ArrayValue.sums, partial(_agg_with_nulls, sum)),
+        (ir.ArrayValue.mins, partial(_agg_with_nulls, min)),
+        (ir.ArrayValue.maxs, partial(_agg_with_nulls, max)),
+        (ir.ArrayValue.means, partial(_agg_with_nulls, statistics.mean)),
+    ],
+    ids=["sums", "mins", "maxs", "means"],
+)
+@notimpl_aggs
+@pytest.mark.parametrize(
+    "data",
+    [
+        param(
+            [[1, 2, 3], [None, 6], [5], [None], [], None],
```

Comment:
Should these be reduced to just [None,6] and [None]? the other cases are covered by the non-null param

Reply:
```suggestion
            [[None, 6], [None]],
```